plugins/cccongress.py

```diff
@@ -202,8 +202,8 @@ def _get_talk(bot, hall, slot=0):
 
     try:
         json_data = _get_json_data(bot)
-    except:
-        """Silently ignore errors"""
+    except Exception as e:
```

Comment:
And this should catch only this custom exception

Reply:
Well it is a good idea. You can add this. But this is a PR for station-plugin, not cccongress.